nsis/tesseract.nsi

```diff
@@ -0,0 +1,1525 @@
+  Section /o "Math / equation detection module" SecLang_equ
+    AddSize 2200
+    !insertmacro Download_Lang_Data equ
+  SectionEnd
+
+  ; The language names are documented here:
+  ; https://github.com/tesseract-ocr/tesseract/blob/master/doc/tesseract.1.asc#languages
+
+  
+  ####ab hier auf tessdata_fast-Pakete aktualisiert####
+  
+  Section /o "Arabic script" SecLang_Arabic
+    AddSize 8880
+    !insertmacro Download_Lang_Data Arabic
+  SectionEnd
+  
+  Section /o "Armenian script" SecLang_Armenian
+    AddSize 7510
+    !insertmacro Download_Lang_Data Armenian
+  SectionEnd
+  
+  Section /o "Bengali script" SecLang_Bengali
+    AddSize 5450
+    !insertmacro Download_Lang_Data Bengali
+  SectionEnd
+  
+  Section /o "Canadian Aboriginal script" SecLang_Canadian_Aboriginal
+    AddSize 6850
+    !insertmacro Download_Lang_Data Canadian_Aboriginal
+  SectionEnd
+  
+  Section /o "Cherokee script" SecLang_Cherokee
+    AddSize 4040
+    !insertmacro Download_Lang_Data Cherokee
+  SectionEnd
+  
+  Section /o "Cyrillic script" SecLang_Cyrillic
+    AddSize 27900
+    !insertmacro Download_Lang_Data Cyrillic
+  SectionEnd
+  
+  Section /o "Devanagari script" SecLang_Devanagari
+    AddSize 17100
+    !insertmacro Download_Lang_Data Devanagari
+  SectionEnd
+  
+  Section /o "Ethiopic script" SecLang_Ethiopic
+	AddSize 8650
+	!insertmacro Download_Lang_Data Ethiopic
+  SectionEnd
+  
+  Section /o "Fraktur script" SecLang_Fraktur
+	AddSize 10400
+	!insertmacro Download_Lang_Data Fraktur
+  SectionEnd
+  
+  Section /o "Georgian script" SecLang_Georgian
+	AddSize 6630
+	!insertmacro Download_Lang_Data Georgian
+  SectionEnd
+  
+  Section /o "Greek script" SecLang_Greek
+	AddSize 2900
+	!insertmacro Download_Lang_Data Greek
+  SectionEnd
+  
+  Section /o "Gujarati script" SecLang_Gujarati
+	AddSize 4780
+	!insertmacro Download_Lang_Data Gujarati
+  SectionEnd
+  
+  Section /o "Gurmukhi script" SecLang_Gurmukhi
+	AddSize 4020
+	!insertmacro Download_Lang_Data Gurmukhi
+  SectionEnd
+  
+  Section /o "Han Simplified script" SecLang_HanS
+	AddSize 5700
+	!insertmacro Download_Lang_Data HanS
+  SectionEnd
+  
+  Section/o "Han Traditional script" SecLang_HanT
+	AddSize 5200
+	!insertmacro Download_Lang_Data HanT
+  SectionEnd
+  
+  Section/o "Han Traditional vertical script" SecLang_HanT_vert
+	AddSize 5200
+	!insertmacro Download_Lang_Data HanT_vert
+  SectionEnd
+  
+  Section/o "Hangul script" SecLang_Hangul
+	AddSize 4620
+	!insertmacro Download_Lang_Data Hangul
+  SectionEnd
+  
+  Section/o "Hangul vertical script" SecLang_Hangul_vert
+	AddSize 4510
+	!insertmacro Download_Lang_Data Hangul_vert
+  SectionEnd
+  
+  Section/o "Hebrew script" SecLang_Hebrew
+	AddSize 4640
+	!insertmacro Download_Lang_Data Hebrew
+  SectionEnd
+  
+  Section/o "Japanese script" SecLang_Japanese
+	AddSize 5610
+	!insertmacro Download_Lang_Data Japanese
+  SectionEnd
+  
+  Section/o "Japanese vertical script" SecLang_Japanese
```

Comment:
`SecLang_Japanese` must be a unique name. It is already used in line 541, so here I'd suggest `SecLang_Japanese_vert`.